url_serde/src/lib.rs

```diff
@@ -0,0 +1,214 @@
+/*!
+
+This crate provides wrappers and convenience functions to make Url and
+Serde work hand in hand.
+
+The supported types are:
+
+* `url::Url`
+
+# How do I use a data type with a `Url` member with Serde?
+
+Use the serde attributes `deserialize_with` and `serialize_with`.
+
+```
+struct MyStruct {
+    #[serde(deserialize_with = "url_serde::deserialize",
```

Comment:
Yes, it does. I would like to add a test case for it but it will need rust nightly until rust 1.15 is released. Is it ok to have a test that run on nightly only?